.github/workflows/release.yaml

```diff
@@ -116,9 +119,9 @@ jobs:
         env:
           CHARMCRAFT_AUTH: ${{ secrets.CHARMHUB_TOKEN }}
         run: |
-          charmcraft upload-resource prometheus-k8s promql-transform-amd64 --file artifacts/promql-transform-amd64
+          charmcraft upload-resource ${{ matrix.charm }} promql-transform-amd64 --file artifacts/promql-transform-amd64
```

Comment:
How does this action behave if `charmcraft resources` doesn't include both `promql-transform-amd64` and `promql-transform-arm64` for the given charm?

Reply:
This will soon be irrelevant, since they'll all get the same `charmcraft-parts` change as Prometheus